packages/components/organisms/f-checkout/test/specs/component/f-checkout-delivery.component.spec.js

```diff
@@ -3,23 +3,22 @@ import CheckoutComponent from '../../../test-utils/component-objects/f-checkout.
 
 describe('f-checkout "delivery" component tests', () => {
     before(() => {
-        browser.url('?path=/story/components-organisms--checkout-component');
-        browser.switchToFrame(0);
+        browser.url('iframe.html?id=components-organisms--checkout-component&knob-Checkout%20Url=%2Fcheckout-delivery.json&knob-Available%20Fulfilment%20Url=%2Fcheckout-available-fulfilment.json&knob-Create%20Guest%20Url=%2Fcreate-guest.json&knob-Get%20Basket%20Url=%2Fget-basket-delivery.json&knob-Auth%20token=a&knob-Login%20Url=%2Flogin&viewMode=story');
         CheckoutComponent.waitForCheckoutComponent();
     });
 
     forEach(['mobileNumber', 'addressLine1', 'addressCity', 'addressPostcode'])
-    .it('each fields error message should be displayed', field => {
+    .it('should display each fields error message', field => {
         // Act
-        CheckoutComponent.clearCheckoutForm();
+        CheckoutComponent.clearCheckoutForm(field);
         CheckoutComponent.goToPayment();
 
         // Assert
         expect(CheckoutComponent.isFieldErrorDisplayed(field)).toBe(true);
     });
 
     forEach(['addressLine1', 'addressLine2', 'addressCity', 'addressPostcode'])
-    .it('address fields should exist', field => {
+    .it('should check if address fields exist', field => {
```

Comment:
How descriptive is the error message if one of these fails? Is it clear enough that you know which field has failed?

Reply:
The error message will tell you exactly which of these it didn't see when you run the tests and this were to fail.